baremetalpxeserver/baremetalpxeserver/pxeserveragent.py

```
@@ -474,6 +476,8 @@ def create_bm_configs(self, req):
         rsp = AgentResponse()
 
         # check preconfiguration md5sum
+        # FIXME: py3 Strings must be encoded before hashing
+        # TODO(py3)
         if hashlib.md5(cmd.preconfigurationContent).hexdigest() != cmd.preconfigurationMd5sum:
```

Ruff (0.15.4)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)